bool planning_result
PlannedAction planned_action
---
bool execution_result
